src/routes/business/businesses.ts

```diff
@@ -0,0 +1,217 @@
+
+  const businesses = ((await findBusinesses()) || []).slice(
+    (number - 1) * size,
+    size * number
+  );
+
+  return res.status(200).send(businesses);
+};
+
+export const updateBusiness = async (req, res) => {
+  // Solaris responds with a 403 when the parameter being updated is "empty",
+  // i.e., `null`. This is not yet implemented here.
+  const fields = [
+    "name",
+    "sector",
+    "industry",
+    "industry_key",
+    "legal_form",
+    "foundation_date",
+    "address",
+    "line_1",
+    "line_2",
+    "postal_code",
+    "city",
+    "state",
+    "country",
+    "tax_information",
+    "tax_country",
+    "tax_confirmation",
+    "registration_number",
+    "registration_issuer",
+    "fatca_relevant",
+    "fatca_crs_confirmed_at",
+    "crs_company_type",
+    "balance_sheet_total",
+    "number_employees",
+    "registration_type",
+    "registration_district",
+    "business_purpose",
+    "terms_conditions_signed_at",
+    "branch",
+    "international_operativity_expectation",
+    "vat_number",
+    "purpose_of_account_opening",
+    "source_of_funds",
+    "expected_annual_revenue",
+    "expected_annual_incoming_funds",
+    "company_status",
+  ];
+
+  const editableFields = [
+    "name",
+    "sector",
+    "industry",
+    "industry_key",
+    "legal_form",
+    "foundation_date",
+    "address",
+    "line_1",
+    "line_2",
+    "postal_code",
+    "city",
+    "state",
+    "country",
+    "tax_information",
+    "tax_country",
+    "tax_confirmation",
+    "registration_number",
+    "registration_issuer",
+    "fatca_relevant",
+    "fatca_crs_confirmed_at",
+    "crs_company_type",
+    "balance_sheet_total",
+    "number_employees",
+    "registration_type",
+    "registration_district",
+    "business_purpose",
+    "terms_conditions_signed_at",
+    "branch",
+    "international_operativity_expectation",
+    "vat_number",
+    "purpose_of_account_opening",
+    "source_of_funds",
+    "expected_annual_revenue",
+    "expected_annual_incoming_funds",
+    "company_status",
+  ];
+
+  const {
+    params: { business_id: businessId },
+    body,
+  } = req;
+  const data = _.pick(body, fields);
+
+  let business;
+  const businessLockKey = `redlock:${process.env.MOCKSOLARIS_REDIS_PREFIX}:business:${businessId}`;
+  await redlock.using([businessLockKey], 5000, async (signal) => {
+    if (signal.aborted) {
+      throw signal.error;
+    }
+    business = (await getBusiness(businessId)) as MockBusiness;
+
+    const fieldsBanned = [];
+    Object.keys(data).forEach((key) => {
+      if (!editableFields.includes(key)) fieldsBanned.push(key);
+    });
+
+    if (fieldsBanned.length) {
+      return res.status(400).send({
+        id: "f0487cda-5a03-11e9-8ebd-02420a86840b",
+        status: 400,
+        code: "deprecated_params",
+        title: "Deprecated Parameters",
+        detail: `Updating ${fieldsBanned[0]} is deprecated.`,
+      });
+    }
+
```

Comment:
the part with ChangeRequest is missing. Similar to person
```
if (isChangeRequestRequired(editable, person)) {
      return createChangeRequest(req, res, person, PERSON_UPDATE, data);
    }
```
maybe it will be already too must for this PR (it will require ui updates too), but then we should create a follow up ticket to not forget it.